### PR TITLE
Cleans up conveyor belts

### LINKED ISSUE #37248: MetaStation cargo shuttle conveyor wrong way around.
The bottom (into-shuttle) conveyor on the shuttle itself is wrong way around (pushing outside) causing crates to bounce back-and-forth between it and the station conveyor.

This is on MetaStation.

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
fix: Fixed a number of conveyor belts leading to the incorrect direction.
fix: Fixed shuttle conveyor belts being out of sync with cargo's conveyors on some maps.
/:cl:

Fixes #37248

Cleans up a ton of dirty conveyor belt vars

Removed all instances of movedir being used, as it was overridden when you turned the belts on anyway. Just set the direction properly damnit.
Removed the convdir switch var, making the switches alot easier to work with. Icons will now be handled based off the switch's dir instead.

Adds directional icons to conveyor belts and conveyor belt switches (only visible in DM) so mappers can easily identify what's going on with them.
Added an inverted subtype for conveyor belts (cuts down on varedits)

Fixed a good number of conveyor belts moving things in the incorrect direction.

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
yet another cargo buff
good

**Comment by Supermichael777** [comment]
This is actually a mapping buff.  Sec auto-processer when?

**Comment by cacogen** [comment]
>buff

**Comment by WJohn** [comment]
Much appreciated. This has always been annoying to work with wihout doing pixel hunting and comparisons to make sure you have the right ones.

**Review by WJohn — DISMISSED** [review]

File: code/modules/shuttle/on_move.dm [hunk 1]
```diff
@@ -266,6 +266,16 @@ All ShuttleMove procs go here
 	if(level==1)
 		hide(T.intact)
 
+/obj/machinery/conveyor/shuttle/afterShuttleMove()
+	. = ..()
+	for(var/obj/machinery/conveyor/C in orange(1, src)) //Automatically assigned direction based on the dir of an adjacent conveyor belt to prevent desync
```
Comment (ninjanomnom) [inline code/modules/shuttle/on_move.dm]: This will also pick up diagonals and potentially fuck up with 2 different direction but parallel conveyor lines.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**